New Areas

PhoenixCodes opened the pull request:
Added the Maim and Pryla'ari forays

## Checklist

The list below serves as a reminder to make sure, all needed changes were added to the repository. You can add a `x` between the square brackets to tick the items when in edit mode or just tick the checkboxes when not.

If you forgot something, don't worry. Just do the changes on the same branches and they are added automatically to the pull request.

- [X] Added changed map file
- [X] Used a descriptive pull request title
- [X] Wrote a short description of the change in the pull request description (see below). Include area names where possible.
- [X] Added a short sentence summarizing the changes to the `changelog.txt` file
- [X] Increased the number in `version.txt` by 1

## Included changes

PhoenixCodes committed:
New Areas

Added the Maim and Pryla'ari forays

ryanstadther commented:
@QwindorRousseau  and i'm not sure why I can't tag vadi, but @PhoenixCodes asked me to take a quick look at this one and it looks great! Two new foray areas! 👍

PhoenixCodes committed:
Added Mirror Tower foray fixes

Resolved Mirror Tower rooms in -1 as well as placed rooms so that people can see how the tower is set up better.

PhoenixCodes commented:
![image](https://user-images.githubusercontent.com/15858451/101289926-479c5200-37b4-11eb-9cfe-3269301a0ad3.png)
Here's the new look I gave Mirror Tower while resolving missing room and wrong area things
